### PR TITLE
chore(docker-compose): add sqlite variable and volumes example

### PR DESCRIPTION
**Pull request by alcaprar**
As also explained [here](https://github.com/photoview/photoview.github.io/pull/25), I was trying `sqlite` (and eventually gave up) and I spend a couple of minutes trying a few configurations.

I hope that this and the doc change can make it more explicit.

(maybe closes https://github.com/photoview/photoview.github.io/pull/25?)

### CODE REVIEW

**Comment by kkovaletp** [comment]
@alcaprar, thanks. I've included your idea in PR #863 and #910 (which is a temporary alternative to master while the project is on hold)

**Review by viktorstrate — APPROVED** [review]

**Review by kkovaletp — APPROVED** [review]
As this fix is included into #863, there is no need to merge this PR, however, I don't see a negative impact of merging it before #863, so I'll approve it

**Pull request merged**
